### DISCUSSION TITLE
Screen noise issue on Galaxy S10

### DISCUSSION THREAD

**Discussion by EllenKim-VCNC** [vimeo/player.js#1001] [Bug reports]
Hi, 
I have a screen noise issue on Galaxy S10.
I was using [react-player](https://www.npmjs.com/package/react-player) 2.9.0 plugin, Next js 12.1.4.
And I inserted the video URL into the component properties. 
refer to the code below please.

```js
     <ReactPlayer
        url={`https://player.vimeo.com/video/${video}`}
        controls={false}
        volume={0}
        loop={loop}
        width="100%"
        height="100%"
        onReady={(player: ReactPlayer) => onReadyVideo(player)}
        onProgress={onVideoProgress}
        onEnded={onEnded}
        playing={playing}
        playsinline
        ref={ref}
        config={{
          vimeo: {
            playerOptions: {
              autoPlay: false,
              controls: false,
              width: 1000,
            },
          },
        }}
      />
```
### Expected Behavior
video without screen noise

### Actual Behavior
Screen noise issue on Galaxy S10, Galaxy Tab.
https://user-images.githubusercontent.com/78350749/173000317-e4237759-68bb-4aa3-93d0-927cfcd14c33.mp4

### Steps to Reproduce
- Check [movelog site](https://www.movelog.online/video?videoId=591536028) using Galaxy S10
- Touch the `click` button

Thanks! =)

**Comment by bdougherty** [comment]
Unless you can reproduce this directly with our player, I'm not sure that we can help here.
